### ISSUE TITLE
Доработки по заявке

### ISSUE THREAD

**Issue by pvs9** [ZZZerock/ivr#1]
Не соблюдена разметка Markdown, ее можно взять из тестовой заявки. В разделе функциональные требования должен быть подробно описан весь функционал. В разделе аналогов нужно провести сравнительный анализ - что будет в вашем приложении, чего нет в аналоге, чего не хватает в аналоге. В рисках не заложены проблемы интеграции с тем же ЭлЖуром, например, уверены, что изучили его API и проблем не возникнет?

**Comment by aakuptsov** [comment]
Присоединяюсь к комментариям Владимира Станиславовича:

1. Файл необходимо оформить в соответствии с требованиями

2. >>У лицеистов нет времени и сил на подсчет своих средних и итоговых оценок.

А зачем тратить силы и время, если ЭлЖур сам прекрасно считает оценки? 

3. >>пользаватель выбирает предмет и оценку, которую хочет получить, программа в свою очередь показывает сколько и каких оценок ему не хватает

Насколько эта информация актуально, если учесть, что по разным предметам разное число КО, ФО, ТО? 

4. В инструментах разработки не нужно указывать "персональный компьютер", это очевидно. Лучше указать средства работы с БД, например, 

5. По сути, проект является монофункциональным. Для зачёта нужно добавить что-то ещё, не только калькулятор оценок. Как вариант, можно попробовать замахнуться на свою версию мобильного приложения для ЭлЖура.

**Comment by ZZZerock** [comment]
Здравствуйте. 1) а что не так у меня с оформлением? 2) В моем приложении не только счет оценок, а также графики и ключевая фишка-калькулятор. 3) я не понял суть вопроса. 5) ваш коллега сказал, что проект будет нормальным и без интеграции с дневником.

**Comment by ZZZerock** [comment]
Изменил инструменты разработки

**Comment by aakuptsov** [comment]
1. Отсутствуют выделенные разделы. Для лучшего понимания нужно сравнить заявку с образцом. 

3. Суть вопроса в следующем: у разных преподавателей одного предмета есть разное число оценок по предмету.  Например, у одного может быть 4 формирующих, у другого - 10. В зависимости от преподавателя ученику можно понадобиться набрать то или иное количество оценок для нужного ему балла. Как это будет учитываться? 

5. >>ваш коллега сказал, что проект будет нормальным и без интеграции с дневником.

Ок, а я говорю, что интеграция нужна. Потому что тогда проект становится очень простым.

**Comment by ZZZerock** [comment]
Здравствуйте, 1) вроде поправил. 2) Это никак не получится обыграть, так как даже Элжуру это не под силу. 3) Смогу ли я вместо интеграции внести другие функции. Я прошу прощения, но еще в октябре я подавал заявку с уверенностью, что ее примут без интеграции, так как изначально мне разрешили, а сейчас, мне возможно придется вообще менять проект

**Comment by aakuptsov** [comment]
>>Это никак не получится обыграть, так как даже Элжуру это не под силу.

Это я в курсе :) Я поэтому и спрашиваю, какая тогда польза от твоего проекта, если этого не будет? Например, продукт мне скажет, что мне надо получить одну творческую до "4", а у преподавателя их ещё три и на самом деле одной явно не хватит.

**Comment by ZZZerock** [comment]
Я не думаю, что моим приложением будут пользоваться в начале четверти или же, до момента, когда не останется пару-тройку работ. Очень часто даже ученики не знают, сколько работ им еще предстоит написать

**Comment by aakuptsov** [comment]
>>так как изначально мне разрешили

Изначально под "разрешить" можно понимать только смысл писать заявку на рассмотрение, это не всегда имеет смысл (например, если заявка будет на сайт-визитку). 

Писать продукт по оценкам без интеграции с ЭлЖуром - в высшей степени странно, откуда тогда брать текущие оценки лицеиста? Вводить их руками, смотря на ЭлЖур? Это очень странно с пользовательской точки зрения. 

PS Просьба в заявке поставить пробелы перед скобками и уточнить разницу между "бал" и "балл".

**Comment by ZZZerock** [comment]
Да, ученик сам будет вписывать оценки. Например, получил оценку, записал. Это будет работать как приложение для подсчета траты денег. Иначе, как приложение будет выдавать еженедельные диаграммы, советы по подготовке к тем или иным предметам.

**Comment by pvs9** [comment]
Вполне себе спокойно, путём выгрузки из ЭлЖура оценок за определенный период и, по необходимости, сохранения их в базу.

**Comment by aakuptsov** [comment]
Присоединяюсь к комментариям Владимира Станиславовича. К защите проектов, связанных с оценками, допускаются только проекты, у которых есть интеграция с ЭлЖуром. Иначе любой проект сводится к решению линейного уравнения.

**Comment by ZZZerock** [comment]
Передумал по поводу интеграции, сделал соответствующие изменение (риски)

**Comment by pvs9** [comment]
Про интеграцию в функциональных требованиях нет ни слова, хотя необходимо обозначить ее наличие, для каких целей

**Comment by ZZZerock** [comment]
Поправил.

**Comment by aakuptsov** [comment]
ЗАЧЁТ, но надо придумать ещё какие-нибудь функции на этапе сценариев, чтобы приложение выглядело солидно.
